mlir/include/mlir/Dialect/SPIRV/Transforms/SPIRVConversion.h

```diff
@@ -202,6 +203,12 @@ SmallVector<int64_t> getNativeVectorShapeImpl(vector::TransposeOp op);
 // For general ops.
 std::optional<SmallVector<int64_t>> getNativeVectorShape(Operation *op);
 
+// Unroll vectors in function signatures to native size.
+bool unrollVectorsInSignatures(Operation *op, MLIRContext *context);
+
+// Unroll vectors in function bodies to native size.
+bool unrollVectorsInFuncBodies(Operation *op, MLIRContext *context);
```

Comment:
In this form it's impossible to tell if `true` indicates success or failure. Return `LogicalResult` instead.

Reply:
Also you can extract context from the op.